models/flair_train.py

```diff
@@ -112,9 +112,9 @@ def train(corpus):
 
 
 if __name__ == "__main__":
-    train_samples = "../data/generated_train_November 12 2019.json"
-    test_samples = "../data/generated_test_November 12 2019.json"
-    val_samples = "../data/generated_validation_November 12 2019.json"
+    train_samples = "../data/train_February_28_2020.json"
```

Comment:
This date is based on the data generated by `Create datasets for Spacy training.ipynb`.